Description: Document the default two-line output in lxc-ls
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: no

Index: lxc/doc/lxc-ls.sgml.in
===================================================================
--- lxc.orig/doc/lxc-ls.sgml.in	2012-08-30 09:48:55.396845000 -0500
+++ lxc/doc/lxc-ls.sgml.in	2012-08-30 10:04:34.779746043 -0500
@@ -57,7 +57,14 @@
     <title>Description</title>
     <para>
       <command>lxc-ls</command> list the containers existing on the
-      system.
+      system.  By default it lists them in two lines.  The first line
+      contains all defined containers, while the second contains all
+      running containers.  If additional arguments are given, they are
+      passed to the 'ls' program.  This can change the output to be
+      multiple lines.  For instance, 'lxc-ls -l' will print a full line
+      for each container.  Since lxc-ls will still print first the
+      existing, then the running containers, running containers will
+      appear to be listed twice.
     </para>
   </refsect1>
 
